chromium/store.js

```diff
@@ -2,21 +2,43 @@
 
 (function(exports) {
 
+let ls;
+
+try{
+  ls = localStorage;
+} catch(e) {
+  ls = {setItem: () => {}, getItem: () => {}};
+}
+
+function initialize(){
+  return new Promise(resolve => {
+    if (chrome.storage.sync) {
+      chrome.storage.sync.set({"sync-set-test": true}, () => {
+        if(chrome.runtime.lastError){
+          setStorage(chrome.storage.local);
+        } else {
+          setStorage(chrome.storage.sync);
+        }
+        resolve();
+      });
+    } else {
+      setStorage(chrome.storage.local);
+      resolve();
+    }
+  });
+}
+
 function setStorage(store) {
   Object.assign(exports, {
     get: store.get,
     set: store.set,
+    localStorage: ls,
+    initialize: initialize
   });
 }
 
-setStorage(chrome.storage.local);
-if (chrome.storage.sync) {
-  chrome.storage.sync.set({"sync-set-test": true}, () => {
-    if(!chrome.runtime.lastError){
-      setStorage(chrome.storage.sync);
-      background.initializeStoredGlobals();
-    }
-  });
-}
+Object.assign(exports, {
```

Comment:
`exports.initialize = initialize;`

Reply:
I'd probably go with:
```javascript
Object.assign.(exports, {initialize})
```
Because you DRY writing the function name. NBD to me either way.